test/client.js

```diff
@@ -483,12 +496,26 @@ describe('Rest Client', function () {
         retries: 2,
         retryTimeout: 0
       });
-      api.put(path, requestBody).reply(500, responseBody);
-      api.put(path, requestBody).reply(500, responseBody);
+      nockRetries(2, {
+        httpMethod: 'put'
+      });
+
       api.put(path, requestBody).reply(201, responseBody);
```

Comment:
Don't think the final response code is necessary. You can just make use of your new `successCode: 201` property.